.github/workflows/Ci.yml

```diff
@@ -15,20 +15,15 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
           cache: npm
 
       - name: 📦 Install dependencies
         run: npm ci
 
       - name: 🧪 Run all tests
-        if: ${{ github.event_name == 'push' }}
         run: npm run test
 
-      - name: 🧪 Run tests for changed files only
```

Comment:
Isn't it a good idea to run the tests for the changed files only? Or is this not compatible?

Reply:
Ideally, you should **always** run **every** test - exactly because of situations like [this](https://github.com/TheAlgorithms/JavaScript/pull/1407#discussion_r1343580070).
And yeah, this means that people should pay some attention that their tests not only run successfully, but do so in a timely manner.